controllers/flowlogspipeline/flp_common_objects.go

```diff
@@ -527,6 +550,42 @@ func (b *builder) addTransformStages(stage *config.PipelineBuilderStage) error {
 		enrichedStage.EncodePrometheus("prometheus", promEncode)
 	}
 
+	if len(topPromMetrics) > 0 && b.info != nil && b.info.AvailableAPIs != nil && b.info.AvailableAPIs.HasSvcMonitor() {
+		topBytesStage := enrichedStage.ExtractTimebased("top_bytes", api.ExtractTimebased{
+			Rules: []api.TimebasedFilterRule{
+				{
+					Name:          "top_src_bytes",
+					IndexKeys:     []string{"SrcK8S_Name", "SrcK8S_Namespace", "SrcK8S_Type", "SrcK8S_HostName", "FlowDirection", "Duplicate"},
+					OperationType: "avg",
```

Comment:
This is the downside of having aggregation done at ingest time: we're setting in stone the operation type.
I think FLP also supports histograms, which would provide more flexibility on queries, but haven't digged into how to use it in the aggregation stage - cc @ronensc iirc you wrote this, right?

Reply:
(but I guess that's not a blocker, we can use avg and see later if there's a need for something else)